src/foam/core/session/PersistentSessionDAO.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * @license
+ * Copyright 2025 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.core.session',
+  name: 'PersistentSessionDAO',
+  extends: 'foam.dao.ProxyDAO',
+  implements: [
+    'foam.core.COREService'
+  ],
+
+  documentation: `The key to 'persistent' sessions is the userId on the
+session. If a user makes a client call with an existing session id and
+a matching user id, then the system considers the user logged in.
+Sessions are NOT-frozen, for reasons discussed elsewhere.  The
```

Comment:
We should move the reasons why into here in the codebase, the github issue may get lost with time.